packages/components/image-viewer/src/index.vue

```diff
@@ -242,6 +242,13 @@ export default defineComponent({
       const { offsetX, offsetY } = transform.value
       const startX = e.pageX
       const startY = e.pageY
+
+      const div = document.getElementsByClassName('el-image-viewer__wrapper')
```

Comment:
You need to use `ref` to get a reference to an element. see: https://v3.vuejs.org/guide/composition-api-template-refs.html

Reply:
@adaex have changed to use `ref`